nx.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
What's this?

Reply:
This is needed in order to handle the cache for the tests. So it runs the tests only if there are differences in the build folder